cardano-db-sync/src/Cardano/DbSync/Util.hs

```diff
@@ -140,29 +134,9 @@ nullMetricSetters =
     , metricsSetDbSlotHeight = const $ pure ()
     }
 
--- The network code catches all execptions and retries them, even exceptions generated by the
--- 'error' or 'panic' function. To actually force the termination of 'db-sync' we therefore
--- need a custom panic function that is guaranteed to abort when we want it to.
--- However, we may not want to abort in production, so we make it optional by use of an
--- environment variable.
```

Comment:
Should we also add this as an option to the NixOS module?

Reply:
I was wondering if it should be a flag rather than a env variable. But that would cause a breaking change so I left it.

Reply:
Why would a flag make it breaking?